app/components/input/ontologies_selector_component.rb

```diff
@@ -0,0 +1,10 @@
+class Input::OntologiesSelectorComponent < ViewComponent::Base
```

Comment:
No need to have a UI component for this, just create a helper that will use the `Input::SelectComponent` UI component.